README.md

```diff
@@ -209,6 +209,19 @@ To build the Ruby code run:
 ```sh
 bazel build //rb/...
 ```
+
+To run tests on a browser:
+
+```sh
+bazel test --cache_test_results=no --test_output=all //rb:<browsername>-test 
```

Comment:
I don't think this is useful without the list of available browsernames. Let's add:

* chrome
* firefox
* firefox-nightly (requires setting ENV["FIREFOX_NIGHTLY_BINARY"])
* safari
* safari-preview
* ie
* edge

Reply:
I was thinking to add the list but seeing other bindings not doing it, I refrained. Will add them

Reply:
Yeah, I think we should be doing it for all the bindings; we need to make bazel usage as obvious as possible.

Reply:
Made the relevant changes. Please have a look.